core-bundle/contao/templates/twig/backend/data_container/table/search_menu.html.twig

```diff
@@ -0,0 +1,38 @@
+{% trans_default_domain "contao_default" %}
+
+{% set select_attributes = attrs()
+    .set('name', 'tl_field')
+    .addClass('tl_select')
+    .addClass('active', active)
+%}
+
+{% set input_attributes = attrs()
+    .set('type', 'search')
+    .set('name', 'tl_value')
+    .set('value', value)
+    .addClass('tl_text')
+    .addClass('active', active)
+%}
```

Comment:
Could you please move these down, so that they are on top of their respective attributes?